src/screens/UserPortal/Volunteer/Groups/Groups.spec.tsx

```
@@ -43,52 +43,105 @@ vi.mock('@mui/icons-material', async () => {
+      const { isOpen, hide } = params;
+      return isOpen
+        ? React.createElement(
+            'div',
+            { 'data-testid': 'volunteerViewModal' },
+            React.createElement('div', null, 'Group Details'),
+            React.createElement(
+              'button',
+              {
+                'data-testid': 'volunteerViewModalCloseBtn',
+                onClick: hide,
+              },
```

Biome (2.3.13)

[error] Provide a valid type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)